houston/generator/rand.py

```diff
@@ -60,7 +60,8 @@ def generate_mission(self):
         if not takeoff:
             raise Exception("No TAKEOFF command found")
         commands = [self.generate_command(takeoff)]
-        for i in range(self.rng.randint(1, self.max_num_commands - 1)):
+        cmds_len = self.rng.randint(1, self.max_num_commands - 1)
+        for i in range(1, cmds_len):
```

Comment:
Why not `range(cmds_len)`?

Reply:
because we always start with `TAKEOFF` so the first command is already taken. But I notice that there's a small bug here. the line before should actually be `cmds_len = self.rng.randint(1, self.max_num_commands)`.